packages/functions/src/config.ts

```diff
@@ -48,4 +50,5 @@ export function registerFunctions(instance: _FirebaseNamespace): void {
       .setServiceProps(namespaceExports)
       .setMultipleInstances(true)
   );
+  instance.registerVersion('functions', version);
```

Comment:
Are you planning to add this to other SDKs in a separate PR?

Reply:
Yes, planning to do a separate PR for every approver group.